model/models.py

```diff
@@ -22,32 +22,55 @@ def __init__(self, edge_index, edge_type, num_rel, params=None):
 		self.p.gcn_dim		= self.p.embed_dim if self.p.gcn_layer == 1 else self.p.gcn_dim
 		self.init_embed		= get_param((self.p.num_ent,   self.p.init_dim))
 		self.device		= self.edge_index.device
+		self.disable_gnn_encoder = self.p.disable_gnn_encoder
 
 		if self.p.num_bases > 0:
 			self.init_rel  = get_param((self.p.num_bases,   self.p.init_dim))
 		else:
 			if self.p.score_func == 'transe': 	self.init_rel = get_param((num_rel,   self.p.init_dim))
 			else: 					self.init_rel = get_param((num_rel*2, self.p.init_dim))
 
-		if self.p.num_bases > 0:
-			self.conv1 = CompGCNConvBasis(self.p.init_dim, self.p.gcn_dim, num_rel, self.p.num_bases, act=self.act, params=self.p)
-			self.conv2 = CompGCNConv(self.p.gcn_dim,    self.p.embed_dim,    num_rel, act=self.act, params=self.p) if self.p.gcn_layer == 2 else None
-		else:
-			self.conv1 = CompGCNConv(self.p.init_dim, self.p.gcn_dim,      num_rel, act=self.act, params=self.p)
-			self.conv2 = CompGCNConv(self.p.gcn_dim,    self.p.embed_dim,    num_rel, act=self.act, params=self.p) if self.p.gcn_layer == 2 else None
+		if not self.disable_gnn_encoder:
+			if self.p.num_bases > 0:
+				self.conv1 = CompGCNConvBasis(self.p.init_dim, self.p.gcn_dim, num_rel, self.p.num_bases, act=self.act, params=self.p)
+				self.conv2 = CompGCNConv(self.p.gcn_dim,    self.p.embed_dim,    num_rel, act=self.act, params=self.p) if self.p.gcn_layer == 2 else None
+			else:
+				self.conv1 = CompGCNConv(self.p.init_dim, self.p.gcn_dim,      num_rel, act=self.act, params=self.p)
+				self.conv2 = CompGCNConv(self.p.gcn_dim,    self.p.embed_dim,    num_rel, act=self.act, params=self.p) if self.p.gcn_layer == 2 else None
 
-		self.register_parameter('bias', Parameter(torch.zeros(self.p.num_ent)))
+			self.register_parameter('bias', Parameter(torch.zeros(self.p.num_ent)))
```

Comment:
```suggestion
		self.register_parameter('bias', Parameter(torch.zeros(self.p.num_ent)))
```
with that fix the the bias should be init again and the error in DistMult shouled be fiexed